EventListener/RouteAnnotationEventListener.php

```diff
@@ -47,14 +48,30 @@ class RouteAnnotationEventListener implements EventSubscriberInterface
      */
     private $defaultSection;
 
+    /**
+     * @var array
+     */
+    private $defaultOptions;
+
     /**
      * @param RouterInterface $router
      * @param string          $defaultSection
+     * @param array           $defaultOptions
      */
-    public function __construct(RouterInterface $router, $defaultSection)
-    {
+    public function __construct(
+        RouterInterface $router,
+        $defaultSection,
+        $defaultOptions = [
```

Comment:
Please don't.
Code should not change that much at once.

Reply:
I'm just adding optional $defaultOptions argument, but row was too long